lib/libguides.tsx

```diff
@@ -21,6 +21,7 @@ const fetchLibGuides = async ({accountId, subjectId}: { accountId?: number, subj
     }
 
     const guidesConfig = {
+      next: {revalidate: 1},
```

Comment:
What is this doing?

Reply:
Aye, but why is it in this PR?

Reply:
:shrug: it was in my local code, so i just committed it with the stuff.

Reply:
Ok, if you want it in then I guess it stays in. 

It would be great if you could write out an RFC-style document that details how the ISR invalidation is intended to work so that I could have something to compare this to. That would also be helpful documentation for the SUP project.